_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.libraries.yml

```diff
@@ -11,7 +11,7 @@ base-theme:
   # This should be all of the style sheets broken up according to smaccs
   css:
     base:
-      css/base/app.css: {}
+      node_modules/rhd-frontend/styles/app.css: {}
```

Comment:
@KamiQuasi is this the correct location now?

Reply:
It is until I update to the latest revisions to the frontend